### PR TITLE
Rename SliverLogicalParentData.scrollOffset to layoutOffset

### PR DESCRIPTION
**Pull request by abarth**
This quantity is actually the layoutOffset of the child, not its scroll offset.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/lib/src/rendering/sliver.dart [hunk 1]
```diff
@@ -545,15 +545,17 @@ class SliverHitTestEntry extends HitTestEntry {
 /// that expect to have many children whose relative positions don't change even
```
Comment (Hixie) [inline packages/flutter/lib/src/rendering/sliver.dart]: the comment higher up is a bit wrong too

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/lib/src/rendering/sliver.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/lib/src/rendering/sliver.dart]: Fixed.

**Pull request merged**